amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/requests/gscore/api/FilePlanAPI.java

```diff
@@ -303,4 +303,31 @@ public HoldCollection getHolds(String filePlanId)
     {
         return getHolds(filePlanId, EMPTY);
     }
+
+    /**
+     * Gets the roles and Capabilities of a file plan.
+     *
+     * @param filePlanId
+     *            The identifier of a file plan
+     * @param parameters
+     *            The URL parameters to add
+     * @return The {Pagination and RoleModel Entries} for the given {@code filePlanId}
+     * @throws RuntimeException
+     *             for the following cases:
+     *             <ul>
+     *             <li>authentication fails</li>
+     *             <li>current user does not have permission to read {@code filePlanId}</li>
+     *             <li>{@code filePlanId} does not exist</li>
+     *             </ul>
+     */
+    public Object getFilePlanRolesAndRelation(String filePlanId, String parameters)
```

Comment:
We should use `RoleCollection` as the return type. Otherwise the records created (e.g `RoleModel`, `GroupModel`) are useless.
Moreover, please add new method to get file plan roles without the `parameters` argument.